### ISSUE TITLE
"UTC" not considered valid IANA time zone identifier

### ISSUE THREAD

**Issue by davidchambers2** [moment/luxon#315]
`'UTC'` is not considered a valid `z` value:

```javascript
> luxon.DateTime.fromFormat('2001-02-03T04:05:06 UTC', "yyyy-MM-dd'T'HH:mm:ss z")
DateTime { Invalid, reason: unparsable }
```

This, though, suggests that `'UTC'` should be considered valid:

```javascript
> new Intl.DateTimeFormat('en-US', {timeZone: 'UTC'}).format()
'8/13/2018'
```

`IANAZone.isValidSpecifier` may be at fault. Do you agree?

**Comment by icambron** [comment]
Hmm, yeah, I'd expect that to work.

**Comment by icambron** [comment]
Fixed in 9552ed7, will be released soon
